docs/integrations/sources/zoho-crm.md

```diff
@@ -0,0 +1,128 @@
+# Zoho CRM
+
+## Sync overview
+
+The Zoho CRM source supports both Full Refresh and Incremental syncs. You can choose if this connector will copy only the new or updated data, or all rows in the tables and columns you set up for replication, every time a sync is run.
+
+Airbyte uses [REST API](https://www.zoho.com/crm/developer/docs/api/v2/modules-api.html) to fetch data from Zoho CRM.
+
+This Source Connector is based on an [Airbyte CDK](https://docs.airbyte.io/connector-development/cdk-python).
+
+### Output schema
+
+This Source is capable of syncing:
+
+* standard modules available in Zoho CRM account
+* custom modules manually added by user, available in Zoho CRM account
+* custom fields in both standard and custom modules, available in Zoho CRM account
+
+The discovering of Zoho CRM module schema is made dynamically based on Metadata API and should generally take no longer than 10 to 30 seconds.
+
+### Note:
+
+Some of Zoho CRM Modules may not be available for sync due to limitations of Zoho CRM Edition or permissions scope. For details refer to the [Scopes](https://www.zoho.com/crm/developer/docs/api/v2/scopes.html) section in the Zoho CRM documentation.
+
```

Comment:
I see that you use dynamic schema and a dynamic list of streams. Please add this info about available streams (all from the `modules` endpoint) to the doc.

Reply:
done